flytekit/clis/sdk_in_container/run.py

```diff
@@ -449,10 +449,14 @@ def to_click_option(
     # If a query has been specified, the input is never strictly required at this layer
     required = False if default_val and isinstance(default_val, ArtifactQuery) else required
 
+    if literal_converter.is_bool():
+        click_cli_paramater_name = f"--{input_name}/--no_{input_name}"
```

Comment:
Can we use `--no-{input_name}`?

Reply:
We can. I opted for `--no_` because `input_name` comes from the python function arguments and is therefore likely to use underscores instead of dashes. On the other hand `--no-` is probably more common amongst general `click` usage. I don't have a strong opinion either way.

Reply:
Please let me know if you would rather I change to `--no-{input_name}`